drivers/pwm/Kconfig.nrf5_sw

```diff
@@ -22,3 +22,24 @@ config PWM_NRF5_SW_0_DEV_NAME
 	help
 	  Specify the device name for the Nordic Semiconductor nRF5x series S/W
 	  PWM driver.
+
+config PWM_NRF5_SW_0_CLOCK_PRESCALER
+	int
+	prompt "Nordic Semiconductor nRF5x series S/W PWM Clock Prescaler"
+	depends on PWM_NRF5_SW
```

Comment:
use IF clause enclosing the DEV_NAME together, instead of using depends on (as the two can be grouped together now)